app/views/content/train-to-be-a-teacher/classroom-placements.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Teacher training school placements"
+heading: "Teacher training school placements"
+subcategory: Postgraduate teacher training
+description: |-
+  Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 
+navigation: 20.20
+navigation_title: School placements
+navigation_description: Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 
+quote:
+  q-giorgio:
+    text: "The highlight of my placements was getting to teach!"
+    name: "Giorgio, biology teacher"
+    classes: quote--indent
+promo_content:
+    - content/train-to-be-a-teacher/promos/mailing-list-promo-itt
+    
+---
+
+School placements are a crucial part of your postgraduate teacher training. Placements are a chance to:
+
+* get experience in the classroom
+* immerse yourself in your placement schools
+* get to know your pupils and colleagues
+* start your career as a teacher
+
+## How long are placements?
+
+School placements last for a minimum of 24 weeks in at least 2 schools. This is to give you a varied and thorough experience in the classroom. 
+
+## Where will my placements take place? 
+
+Your training provider will select placement schools for you. They will contact you and discuss your situation to help them select a location that you can travel to. 
+
+## What will I do on my placement?  
+
+You school placement aims to give you enough experience to: 
+
+* meet all the teachers’ standards 
+
+* achieve qualified teacher status (QTS)  
+
+So, you’ll be spending a lot of time in the classroom learning how to teach lessons. Beyond the classroom, you can also get involved in other aspects of school life, like school trips, school plays, parents' evenings and staff meetings.  
+
+Over your placement you’ll get to see your pupils progress and get an insight into how rewarding teaching can be.  
+
+$q-giorgio$
+
+## What support will I get during my placement? 
+
+You’ll start your placements well equipped with a toolkit of theory and techniques that you'll learn in training. Placements are a chance to put that into practice.  
+
+You’ll also have support from mentors and your school leadership throughout your placements. For example, when you start your placement, you might start by observing lessons and discuss with your mentor when you feel comfortable to start teaching. 
+
+Your placement school may also have documents and policies in place to support their teachers. For example, behaviour policies.  
```

Comment:
The second sentence feels a little bit odd (short)?

Also - **all** schools need a behaviour policy (https://www.gov.uk/school-behaviour-exclusions), so the `may` feels a bit odd if we are just using behaviour policy as an example of docs/policies a school **may** have?

Something to discuss with policy as to how this needs to be worded?